haproxystats/process.py

```diff
@@ -554,6 +554,7 @@ def process_backends(self, data_frame, filter_backend):
         log.debug('processing statistics for backends')
         # Filtering for Pandas
         is_backend = data_frame['svname_'] == 'BACKEND'
+        got_traffic = data_frame['lbtot'] > 0
```

Comment:
Can we please have a comment here why we are doing this?

Reply:
Added.